src/big/big_decimal.cr

```diff
@@ -66,19 +67,19 @@ struct BigDecimal < Number
     end
   end
 
-  # Creates an new `BigDecimal` from `Int`.
+  # Creates a new `BigDecimal` from `Int`.
   def initialize(num : Int)
     initialize(num.to_big_i, 0)
   end
 
-  # Creating a `BigDecimal` from `Float`.
+  # Creates a new `BigDecimal` from `Float`.
   #
   # NOTE: Floats are fundamentally less precise than BigDecimals, which makes initialization from them risky.
   def initialize(num : Float)
     initialize num.to_s
   end
 
-  # Creates a new `BigDecimal` from `BigInt`/`UInt64`, which matches the internal representation.
+  # Creates a new `BigDecimal` from `BigInt` / `UInt64`, which matches the internal representation.
```

Comment:
Isn't this doc comment wrong, it's `` `BigInt` / 10 ** `UInt64` ``.

Reply:
I don't think so, since method definition is just:

```cr
# Creates a new `BigDecimal` from `BigInt` / `UInt64`, which matches the internal representation.
def initialize(@value : BigInt, @scale : UInt64)
end
```

Reply:
Yeah, Sija is right there.

Reply:
I meant you should note that the value this represents is `@value * (10 ** @scale)`.

Reply:
I'm confused... From my understanding of this doc comment, it states each argument types respectively (`@value : BigInt` + `@scale : UInt64`), and your note seems to cover the case for `initialize(value : UInt64)`.

Reply:
I'm talking about documenting how the internal representation works before exposing a constructor to the public.

Reply:
I think you meant `@value / 10 ** @scale`, not `@value * 10 ** @scale`.
Also I don't think it is how *internal representation works*, it's relevant only when converting `BigDecimal` to `Int`.

Reply:
I think how  those two numbers encode the number this represents does matter. We should document it *somewhere*.

At least use "`BigInt` and `UInt64`" instead of "`BigInt` / `UInt64`".

Reply:
Wording changed, is there anything else blocking this PR from being merged?